private/projects/pdk-nag-project.ts

```diff
@@ -14,15 +14,30 @@ export class PDKNagProject extends PDKProject {
       outdir: "packages/pdk-nag",
       defaultReleaseBranch: "mainline",
       sampleCode: false,
-      jest: false,
+      jest: true,
       name: "pdk-nag",
       depsUpgrade: false,
       peerDeps: ["aws-cdk-lib", "constructs", "cdk-nag"],
-      devDeps: ["cdk-nag"],
+      devDeps: [
+        "cdk-nag",
+        "@aws-cdk/assert",
+        "@types/mustache",
+        "mustache",
+        "@types/fs-extra",
+        "fs-extra",
+      ],
       author: "AWS APJ COPE",
       authorAddress: "apj-cope@amazon.com",
       repositoryUrl: "https://github.com/aws/aws-prototyping-sdk",
       stability: Stability.STABLE,
     });
+    this.packageTask.exec("ts-node scripts/generate-pack-markdown.ts");
+    this.tsconfigDev.addInclude("scripts/**/*.ts");
```

Comment:
Change this to `this.tsconfigEslint!.addInclude("scripts");`

Reply:
Also please verify that the .mustache file is packaged in the bundled tar.gz file in dist directory.